templates/terraform/modules/environment/variables.tf

```diff
@@ -80,6 +80,12 @@ variable "vpc_use_single_nat_gateway" {
   default     = true
 }
 
+variable "vpc_nat_instance_types" {
+   description = "Candidates of instance type for the NAT instance"
+   type        = list
+   default     = ["t3.nano", "t3a.nano"]
```

Comment:
Let's just provide t3.nano here. It's a bit confusing having 2 types here since we're not using spot instances.